src/FakeItEasy/A.cs

```diff
@@ -75,6 +107,18 @@ public static T Dummy<T>()
         }
 
         /// <summary>
+        /// Gets a dummy object of the specified type. The value of a dummy object
+        /// should be irrelevant. Dummy objects should not be configured.
+        /// </summary>
+        /// <param name="typeOfDummy">The type of dummy to return.</param>
+        /// <returns>A dummy object of the specified type.</returns>
+        /// <exception cref="ArgumentException">Dummies of the specified type can not be created.</exception>

```

Comment:
It's odd that this method seems to be more accessible than `Dummy<T>` (`EditorBrowsable`). I'd think they should be the same.